emukit/bayesian_optimization/loops/bayesian_optimization_loop.py

```diff
@@ -48,3 +54,12 @@ def __init__(self, model: IModel, space: ParameterSpace, acquisition: Acquisitio
         loop_state = create_loop_state(model.X, model.Y)
 
         super().__init__(candidate_point_calculator, model_updaters, loop_state)
+
+    def get_current_best(self) -> dict:
+        """
+        Returns a dictionary with the current best found location (minimum) and its value.
+
+        """
+        return {'minimum': self.loop_state.X[np.argmin(self.loop_state.Y),None], 'value':np.min(self.loop_state.Y)}
```

Comment:
I'd use `:` rather than `None` as it makes sense of the minimum location to be 1d rather than 2d.

It's not clear "minimum" is the location of the minimum. Maybe "minimum_location" and "minimum_value" are clearer key name?

Reply:
Done